src/System.Threading.ThreadPool/tests/ThreadPoolTests.netcoreapp.cs

```diff
@@ -188,5 +189,140 @@ private sealed class InvalidWorkItemAndTask : Task, IThreadPoolWorkItem
             public InvalidWorkItemAndTask(Action action) : base(action) { }
             public void Execute() { }
         }
+
+        [Fact]
+        public void MetricsTest()
+        {
```

Comment:
Should RemoteExecutor.Invoke be used to have this run in a separate process so that the metrics aren't influenced by other tests running concurrently?